#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_nosuse.dpatch by Petter Reinholdtsen
##
## DP: Purpose: Disable the SUSE flag.
## DP: Fixes:   None, made as part of Debian porting.
## DP: Status:  Debian specific, showed to upstream.

@DPATCH@
--- a/Makefile
+++ b/Makefile
@@ -9,7 +9,7 @@
 #DESTDIR =	/tmp/root
 #DEBUG	 =	-DDEBUG=1 -Wpacked
 DEBUG	 =
-ISSUSE	 =	-DSUSE
+ISSUSE	 =	-DNOTSUSE
 DESTDIR	 =
 VERSION	 =	1.11.10
 DATE	 =	$(shell date +'%d%b%y' | tr '[:lower:]' '[:upper:]')
